src/main/java/com/tabgok/entity/Machine.java

```diff
@@ -0,0 +1,54 @@
+package com.tabgok.entity;
+
+import com.tabgok.data.StringData;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * The machine is a model of a machine.  Importantly, the machine is the top-level
+ * model for a machine, meaning it contains within it either references to, or values of,
+ * everything within a machine from Hardware to Filesystems to applications.
+ */
+public class Machine extends Entity {
+    private StringData MachineID;
```

Comment:
Oracle's style guide has a large warning saying "hasn't been updated since 1999" (http://www.oracle.com/technetwork/java/codeconvtoc-136057.html).  Should I look there or here: https://google.github.io/styleguide/javaguide.html or somewhere else?